packages/storybook/stories/va-breadcrumbs-uswds.stories.jsx

```diff
@@ -108,6 +108,49 @@ const WrappingCrumbsTemplate = ({
   );
 };
 
+
+const WithRouterTemplate = ({
+  label,
+  'disable-analytics': disableAnalytics,
+  uswds,
+  wrapping
+}) => {
+  const breadcrumbs = [
+    { label: 'Home', href: '/#1' },
+    { label: 'Federal Contracting', href: '/#2', isRouterLink: true },
+    { label: 'Contracting assistance programs', href: '/#3', isRouterLink: true },
+    {
+      label: 'Women-owned small business federal contracting program',
+      href: '/#4',
+      isRouterLink: true
+    },
+  ];
+
+  function handleRouteChange({detail}) {
+    const { href } = detail;
+    console.log(`the href is: ${href}`)
+  }
+
+  return (
+    <div>
+      <p>Some of the breadcrumbs in this example have a <code>isRouterLink:true</code> property.
+        When the corresponding anchor tag is clicked these links emit a <code>route-change</code> event.
+        This event can be handled in a React component where utilities provided 
+        by React Router can be used to change the page under view.
+      </p>
```

Comment:
I wonder if this example could be clearer with a small code block of what the callback could look like. Maybe similar to how we're explaining how to handle custom validation on memorable date?

https://2188-poc-ii--60f9b557105290003b387cd5.chromatic.com/?path=/docs/components-va-memorable-date--custom-validation

Reply:
@jami - good call, done.